compiler/one-cmds/onelib/utils.py

```diff
@@ -122,6 +122,17 @@ def parse_cfg(args, driver_name):
             raise AssertionError('configuration file must have \'' + section_to_run +
                                  '\' section')
 
+        if section_to_run == 'one-optimize':
+            # Q. Why we handle default options first?
+            # A. We allow users to overwrite individual options. For example,
+            #    if O1=True and FoldDequantize=False, FoldDequantize is set to False.
```

Comment:
👍🏻

Reply:
Strictly speaking, it would work even though this if condition moves into below L145. But, looks good to me:)